File: Colibri.Survey/Survey.Spa/ClientApp/app/shared/models/form-builder/survey.model.ts

import { PageModel } from './page.model';

export class SurveyModel {

    id: string;
    name: string;
    description: string;
    pages: PageModel[];
    isShowDescription: boolean;
    isOpenAccess: boolean;
    isShowProcessCompletedText: boolean;
    isLocked: boolean;
    processCompletedText: string;
    constructor(options: any = {}) {
        this.id = options['id'];
        this.name = options['name'] || '';
        this.description = options['description'] || '';
        this.pages = options['pages'];
        this.isShowDescription = options['isShowDescription'] || true;
        this.isShowProcessCompletedText = options['isShowProcessCompletedText'] || true;
        this.processCompletedText = options['processCompletedText'] || '';
        this.isOpenAccess = options['isOpenAccess'] || true;
        this.isLocked = options['isLocked'] || true;
    }
}

File: Colibri.Survey/Survey.ApplicationLayer/Dtos/Models/SurveyModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Survey.ApplicationLayer.Dtos.Models
{
    public class SurveyModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsShowDescription { get; set; }
        public bool IsShowProcessCompletedText { get; set; }
        public string ProcessCompletedText { get; set; }
        public bool IsOpenAccess { get; set; }
        public DateTime DateCreated { get; set; }
        public IEnumerable<PageModel> Pages { get; set; }
    }
}
